README.md

```diff
@@ -397,6 +397,7 @@ The default values for each rule are described below.
 | no_array_responses           | error   |
 | parameter_order              | warning |
 | operation_id_naming_convention | warning |
+| check_required_properties    | warning |
```

Comment:
The naming convention for rules is usually to describe what the issue is. For example, `no_array_responses` leads to `error`.

I think we should name this something like `undefined_required_properties`

Reply:
Also there's one more place in the README that needs to be filled in - the description for each rule